From d474e9a51d5af20541f2e5da03b433266363cd35 Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Sun, 27 Jul 2014 12:41:34 +0200
Subject: [PATCH] Fix man page installation directory

Use the right automake invocation to make the man page properly go to
/usr/share/man/man1.
---
 data/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index a418caf..b2a4a45 100644
--- data/Makefile.am
+++ data/Makefile.am
@@ -11,8 +11,7 @@ appdata_in_files = corebird.appdata.xml.in
 appdata_DATA = $(appdata_in_files:.xml.in=.xml)
 @INTLTOOL_XML_RULE@
 
-manpagedir = $(mandir)
-manpage_DATA = corebird.1
+man_MANS = corebird.1
 
 if ENABLE_CATALOG
   catalogdir = $(prefix)/share/glade/catalogs/
